packages/grafana-runtime/src/utils/analytics.ts

```diff
@@ -43,13 +43,20 @@ export const reportPageview = () => {
  * @public
  */
 export const reportInteraction = (interactionName: string, properties?: Record<string, any>) => {
-  getEchoSrv().addEvent<InteractionEchoEvent>({
-    type: EchoEventType.Interaction,
-    payload: {
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
I found that quite handy to test locally event emitting (see screenshot).

But:
1) there could be issues from that change I have not foreseen (we could mitigate by removing the `else` to force the `addEvent` to be called?
2) there is very possibly another way to locally test `reportInteraction`?

Reply:
would it make sense to flip the logic and explicitly only send the event if we are running in prod? At the moment, this will send the event even if we are running in `test` and I'm assuming we don't want that?

Looks like we don't log in `prod` or `test` here either: https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/utils/logger.ts#L25